assets/js/graphs.js

```diff
@@ -42,14 +50,63 @@ function displayLineChart(id, values) {
 
     // Display the chart
     const chart = new Chart(document.getElementById(id), {
-        type: "bar",
+        type: type,
         data,
         options
     })
 
     return chart
 }
 
+function displayCountryChart(id, values, charts) {
+
+    fetch('https://unpkg.com/world-atlas/countries-50m.json').then((r) => r.json()).then((data) => {
```

Comment:
I'd rather not have Shika rely on some random URL existing, can we instead import world-atlas as package?